src/pfe/portal/routes/projects/remoteBind.route.js

```diff
@@ -328,6 +361,23 @@ async function syncToBuildContainer(project, filesToDelete, modifiedList, timeSt
   }
 }
 
+// List all the directories under a given directory
+async function recusivelyListDirectories(absolutePath, relativePath = '') {
```

Comment:
I think `recusivelyListDirectories` and `listFilesInDirectory` should probably be the same function and should definitely use the same naming scheme. (Probably `recusivelyListDirectories` and `recusivelyListFiles` otherwise we'll have `listDirectoriesInDirectory`.)

Reply:
Also it's recursively not recusively. ;-)

Reply:
I've renamed this function and removed the listFilesInDirectory one.